tests/test_bitround.py

```diff
@@ -74,3 +76,32 @@ def test_bitround_xarray_julia_equal(air_temperature, dtype, keepbits):
         ds_xr_bitrounded = xb.xr_bitround(ds, keep)
         ds_jl_bitrounded = xb.jl_bitround(ds, keep)
         assert_equal(ds_jl_bitrounded, ds_xr_bitrounded)
+
+
+def test_bitround_along_dim_keepbits():
+    ds = xr.Dataset(
+        {"air": (("lat", "lon", "time"), np.random.rand(3, 3, 3))},
```

Comment:
`np.random.rand` shouldn't be used in a test without a [random seed](https://en.wikipedia.org/wiki/Random_seed) as the results won't otherwise be deterministic. Can't you just use the `air_temperature` dataset we are already using in these tests?